mods/common/languages/chrome/en.ftl

```diff
@@ -151,9 +151,9 @@ checkbox-debug-panel-show-screenmap = Show Screen Map
 label-mission-objectives = Mission:
 
 ## ingame-infoscripterror.yaml
-label-script-error-panel-desca = The map script has encountered a fatal error
-label-script-error-panel-descb = The details of the error have been saved to lua.log in the logs directory.
-label-script-error-panel-descc = Please send this file to the map author so that they can fix this issue.
+label-script-error-panel-desc-a = The map script has encountered a fatal error.
```

Comment:
Why are you adding a full stop here but removing it on many other locations?

Reply:
See guidelines

Reply:
Which guidelines? It feels kinda random actually.

Reply:
By that I meant the guidelines at the top of the PR to leave fullstops. I added a lot of them back in, I’ll ask Pun if that fits his criteria.